docs/cloud-resolver.md

```diff
@@ -52,23 +65,37 @@ def train_model(...):
 
 `inline=False` functions will be executed asynchronously as separate Kubernetes pods.
 
-## Customize resource requirements
+### Customize resource requirements
 
 Sematic lets you customize what resources to allocate to particular pipeline
 steps (Sematic Functions).
 
 Pass a `ResourceRequirements` object to the Sematic decorator as follows:
 
 ```python
-@sematic.func(
-    inline=False,
-    resource_requirements=ResourceRequirements(
-        kubernetes=KubernetesResourceRequirements(
-            node_selector={"node.kubernetes.io/instance-type": "g4dn.xlarge"},
-            requests={"cpu": "2", "memory": "4Gi"},
-        )
+from sematic import ResourceRequirements, KubernetesResourceRequirements
+
+GPU_RESOURCE_REQS = ResourceRequirements(
+    kubernetes=KubernetesResourceRequirements(
+        # Note: the kind of node selector options that are valid will depend on
+        # your particular deployment of Kubernetes. Talk to the person who manages
+        # your Kubernetes cluster if you think you might need this. It is primarily
+        # useful in Sematic to gain access to nodes with GPUs.
+        node_selector={"node.kubernetes.io/instance-type": "g4dn.xlarge"},
+
+        # The resource requirements of the job. Information on the format of valid
+        # values can be found here: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
+        # the dictionary provided here will be used for both "limits" and "requests".
+        requests={"cpu": "1", "memory": "1Gi"},
+
+        # By default, Docker uses a 64MB /dev/shm partition. If this flag is set,
+        # a memory-backed tmpfs that expands up to half of the available memory file
+        # is used instead.
+        mount_expanded_shared_memory=True,
```

Comment:
This feature, as well as the inline feature below, was in an unreachable file. Moved here and deleted the original file.